luminex/src/org/labkey/luminex/view/guideSetConfirmDelete.jsp

```diff
@@ -60,7 +61,7 @@
             <br>
             Type: <% if(gs.isValueBased()) out.print("Value-based"); else out.print("Run-based"); %>
             <br><br>
-            Current Guide Set: <%=gs.getCurrent()%>
+            Current Guide Set: <%=HtmlString.of(gs.getCurrent().toString())%>
```

Comment:
I would just call h() on Booleans. It's simpler... plus it handles the case where gs.getCurrent() is null. (This code will NPE.)

Reply:
We could consider adding an h(Boolean) variant to JspBase, but I think the h(Object) variant is essentially the same thing.

Reply:
Good point!